docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.mdx

```diff
@@ -44,16 +44,16 @@ To write a `package.json` file with default values, run `npm init -y` instead. D
 
 ## Install npm packages
 
-1. Install `big-design`, `next`, `react`, `react-dom`, and `styled-components`.
+1. Install `big-design`, `big-design-icons`, `big-design-theme`, `dotenv`, `next`, `react`, `react-dom`, and `styled-components`.
```

Comment:
`big-design-icons`, `big-design-theme`, and `dotenv` packages are not needed at step 1 - their use doesn't come until later, specifically steps 3 and 4

Reply:
the development server won't start without dotenv, and the user starts it at the end of step 1. 

additionally, the other big-design package is installed in this step. We don't currently have a step to install packages in step 4, so I'm going to leave this in pending a future rewrite.